shell/client/setup-wizard/wizard.html

```diff
@@ -44,7 +44,7 @@
                                    mayJumpTo=(mayJumpTo "identity")
                                    route="setupWizardIdentity"
                                    className=itemClassName}}
-      Login provider(s)
+      Identity providers
```

Comment:
For your specific question: I think calling them "Login providers" is OK even if they create "identities" for users.

In general: I'm +1 on separating this discussion into a separate ticket/PR. That way, we can have a conversation that isn't a blocker on an existing PR. For my part, I don't need the perfect to be the enemy of the good here.